ultraplot/axes/plot.py

```diff
@@ -1067,6 +1068,65 @@
     edgefix="\n%(axes.edgefix)s",  # noqa: E501
 )
 
+_graph_docstring = r"""
+Plot a networkx graph with flexible node, edge, and label options.
+
+Parameters
+----------
+g : networkx.Graph
+    The graph object to be plotted. Can be any subclass of `networkx.Graph`, such as
+    `networkx.DiGraph` or `networkx.MultiGraph`.
+layout : callable or dict, optional
+    A layout function or a precomputed dict mapping nodes to 2D positions. If a function
+    is given, it is called as ``layout(g, **layout_kw)`` to compute positions.
+layout_kw : dict, default: {}
+    Keyword arguments passed to the layout function, if `layout` is callable.
+node_kw : dict, default: {}
+    Additional keyword arguments passed to the node drawing function. These can include
+    size, color, edgecolor, cmap, alpha, etc., depending on the backend used.
+edge_kw : dict, default: {}
+    Additional keyword arguments passed to the edge drawing function. These can include
+    width, color, style, alpha, arrows, etc.
+label_kw : dict, default: {}
+    Additional keyword arguments passed to the label drawing function, such as font size,
+    font color, background color, alignment, etc.
+labels : bool or iterable, default: False
+    Whether to show node labels. If `True`, labels are drawn using node names. If an
+    iterable is given, only those nodes are labeled.
+nodes : bool or iterable, default: True
+    Which nodes to draw. If `True`, all nodes are drawn. If an iterable is provided, only
+    the specified nodes are included.
+edges : bool or iterable, default: True
+    Which edges to draw. If `True`, all edges are drawn. If an iterable of edge tuples is
+    provided, only those edges are included.
+grid : bool, default: False
+    Whether to show a background grid.
+aspect : {'equal', 'auto'} or float, default: 'equal'
+    The aspect ratio of the plot. `'equal'` ensures that the units are the same in every direction.
+    `'auto'` lets the axes scale independently. A float can specify a custom aspect ratio.
```

Comment:
Same with this one. People should be using `ax.format` and we should not provide more than one obvious way to do things.